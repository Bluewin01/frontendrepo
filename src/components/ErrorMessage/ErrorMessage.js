import React from "react";
import { Form } from "react-bootstrap";
import "../../assets/css/editor.css";

export const ErrorMessage = (props) => {
  return (
    <>
      {props.error && (
        <Form.Label data-testid="errorMessage" className="errorMessage">
          {props.error}
        </Form.Label>
      )}
    </>
  );
};
