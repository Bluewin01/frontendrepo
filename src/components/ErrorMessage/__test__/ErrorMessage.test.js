import React from "react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import { ErrorMessage } from "../ErrorMessage";

describe("Component: Error Message", () => {
  it("error message component renders correctly", () => {
    const tree = renderer.create(<ErrorMessage></ErrorMessage>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
