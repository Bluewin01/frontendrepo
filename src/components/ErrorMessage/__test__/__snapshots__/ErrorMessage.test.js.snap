// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Error Message error message component renders correctly 1`] = `null`;
